Forta-Bots-Challenge-3/README.md

```diff
@@ -0,0 +1,27 @@
+# MakerDAO Bridge Invariant Bot
+
+## Description
+
+This bot detects when the MakerDAO bridge invariant is violated on Arbitrum and Optimism
+
+## Supported Chains
+
+- Ethereum
+- Arbitrum
+- Optimism
+
+## Alerts
+
+- NETHERMIND-1
```

Comment:
I think there could be a more descriptive name. I'll let you decide, but one that hints at the logic of the bot.